lib/matplotlib/contour.py

```diff
@@ -668,22 +668,18 @@ def _find_closest_point_on_leg(p1, p2, p0):
 
     # handle degenerate case
     if np.all(p2 == p1):
-        d = np.sum((p0 - p1)**2)
+        d = np.sum((p0 - p1) ** 2)
```

Comment:
I'd rather not place spaces around the `**` operator because it reduces readability. This seems to be a generally accepted code style https://github.com/PyCQA/pycodestyle/issues/292

Reply:
I don't really agree *in this case* because (due to the parentheses) `**` ends up having a lower precedence than `-`, which the current spacing doesn't show.
OTOH I don't want to start a pep8 exegesis here so I just reverted the change.